src/main/java/net/sf/jabref/model/database/BibDatabase.java

```diff
@@ -562,4 +554,8 @@ public void setEpilog(String epilog) {
     public String getEpilog() {
         return epilog;
     }
+
+    public EventBus getEventBus() {
```

Comment:
I'm not sure whether we should really do information hiding here. I think, we also want to publish changes on the bus. Maybe, if the entry editor changes something?

Possibly, its the best way not to expose the event bus here, but expose it when working on the eventBus for a specific entry.

Currently, I don't know what requirements we have because of #970.